README.md

```diff
@@ -240,7 +240,7 @@ name: GitHub Pages
 on:
   push:
     branches:
-      - master  # Set a branch to deploy
```

Comment:
Since this is a workflow example for the docsy-example repository, it seems that there is no problem with the master as it is. Actually, the Git default branch of the docsy-example still seems to be the master.

https://github.com/google/docsy-example

Reply:
> Since this is a workflow example for the docsy-example repository, it seems that there is no problem with the master as it is. Actually, the Git default branch of the docsy-example still seems to be the master.
> 
> https://github.com/google/docsy-example

You are right, there is only a pending [ticket](https://github.com/google/docsy-example/issues/151) for the branch rename. Reverted my change.